### ISSUE TITLE
"Remove follower" does not work due to referencing wrong Follow ID

### ISSUE THREAD

**Issue by trwnh** [syuilo/Misskey#9250]
## 💡 Summary

"Remove follower" does not work due to wrong Follow ID

## 🥰 Expected Behavior

A Reject Follow is sent out referencing the remote Follow ID, or null ID for transient Follow activity

## 🤬 Actual Behavior

A Reject Follow is sent out referencing the *local* Follow ID

## 📝 Steps to Reproduce

1. Try to Remove follower

## 📌 Environment

Misskey version: 12.119.0

![image](https://user-images.githubusercontent.com/10606431/205129647-be8eb5ca-3d6b-4d26-b72a-ce27925ea69c.png)

Simplified representation of the activity:

```yaml
id: misskey.bubbletea.dev/1153...
actor: misskey.bubbletea.dev/users/8r0...
type: Reject
object:    # local pseudo-id
    id: misskey.bubbletea.dev/follows/97hb...
    actor: akkoma.trwnh.com/users/a
    type: Follow
    object: misskey.bubbletea.dev/users/8r0...
```

Akkoma fails to recognize the Follow by its remote ID. This is a bug in Akkoma, but Misskey should probably send remote/original Follow ID if it is available. Otherwise `id` should not be included.

Acceptable solution:

```yaml
id: misskey.bubbletea.dev/1153...
actor: misskey.bubbletea.dev/users/8r0...
type: Reject
object:    # no id
    actor: akkoma.trwnh.com/users/a
    type: Follow
    object: misskey.bubbletea.dev/users/8r0...
```

Also acceptable solution:

```yaml
id: misskey.bubbletea.dev/1153...
actor: misskey.bubbletea.dev/users/8r0...
type: Reject
object:    # original id on remote domain
    id: akkoma.trwnh.com/...
    actor: akkoma.trwnh.com/users/a
    type: Follow
    object: misskey.bubbletea.dev/users/8r0...
```

**Comment by shibaobun** [comment]
Just a note that this bug may also result in being unable to refollow that same user later. With 13.4.0, I'm currently unable to refollow users on pleroma, which I think is due to this bug

**Comment by trwnh** [comment]
not fixed by #10600. "remove follower" still does not work due to id of inner Follow when sending out Reject Follow
